File: app/services/omni/allocation_detail.rb
class Omni::AllocationDetail::Service
  include Buildit::Service::Base

  service 'AllocationDetail', 'Omni.service'

  connected_mode(Buildit::Service::Backend::Crud) do |config|
    config.model = Omni::AllocationDetail
  end

end # class Omni::AllocationDetail::Service

File: app/assets/javascripts/omni_desktop/store/AllocationDetail.js
Ext.define('Omni.store.AllocationDetail', {
  extend       : 'Ext.data.Store',
  alias        : 'store.omni-AllocationDetail',
  model        : 'Omni.model.AllocationDetail',
  autoLoad     : false,
  storeId      : 'AllocationDetailStore',
  remoteFilter : true,
  remoteSort   : true,

  constructor  : function (config) {
    var me    = this;
    me.callParent(this);
    
    var proxy = Ext.Object.merge({}, me.getProxy());
    me.setProxy(proxy);
  } // constructor

}); // Ext.define('Omni.store.AllocationDetail'